src/Metadata/AbstractContainer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\Image\Metadata;
+
+use Contao\Image\Exception\InvalidImageContainerException;
+use Contao\Image\Exception\InvalidImageMetadataException;
+
+abstract class AbstractContainer
```

Comment:
I think I would prefer an interface `ContainerInterface` and have the abstract implement it. This would IMHO make it a bit cleaner when adding your own containers/formats.

Reply:
I’d go with `ImageContainerInterface`